src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -140,6 +140,8 @@
 
     public int nextSubnet;
 
+    private String minUpHours;
```

Comment:
This should be `Integer` for a nullable value, or `int` for a non-null value. Type conversion is handled automatically for you at the point of `@DataBoundConstructor` and `@DataBoundSetter`, so you can maintain the real type throughout your code.